packages/langium/src/services.ts

```diff
@@ -43,12 +44,14 @@ export type LangiumLspServices = {
     Connection?: Connection
     DocumentHighlighter: DocumentHighlighter
     DocumentSymbolProvider: DocumentSymbolProvider
+    FoldingRangeProvider: FoldingRangeProvider
     GoToResolver: GoToResolver
     ReferenceFinder: ReferenceFinder
 }
 
 export type LangiumDefaultServices = {
     parser: {
+        MultilineComment: string
```

Comment:
I'd propose to put this into a object like this (naming can be discussed):
```typescript
export interface GrammarConfig {
    multilineCommentRules: string[]
}
```

Reply:
Sure, makes sense.